common/reviews/api/node-core-library.api.ts

```diff
@@ -1,3 +1,75 @@
+// @beta (undocumented)
+class Colors {
+  // (undocumented)
+  static black(text: string | IColorableSequence): IColorableSequence;
+  // (undocumented)
+  static blackBackground(text: string | IColorableSequence): IColorableSequence;
+  // (undocumented)
+  static blue(text: string | IColorableSequence): IColorableSequence;
+  // (undocumented)
+  static blueBackground(text: string | IColorableSequence): IColorableSequence;
+  // (undocumented)
+  static cyan(text: string | IColorableSequence): IColorableSequence;
+  // (undocumented)
+  static cyanBackground(text: string | IColorableSequence): IColorableSequence;
+  // (undocumented)
+  static gray(text: string | IColorableSequence): IColorableSequence;
+  // (undocumented)
+  static grayBackground(text: string | IColorableSequence): IColorableSequence;
+  // (undocumented)
+  static green(text: string | IColorableSequence): IColorableSequence;
+  // (undocumented)
+  static greenBackground(text: string | IColorableSequence): IColorableSequence;
+  // (undocumented)
+  static magenta(text: string | IColorableSequence): IColorableSequence;
+  // (undocumented)
+  static magentaBackground(text: string | IColorableSequence): IColorableSequence;
+  // (undocumented)
+  static red(text: string | IColorableSequence): IColorableSequence;
+  // (undocumented)
+  static redBackground(text: string | IColorableSequence): IColorableSequence;
+  // (undocumented)
+  static white(text: string | IColorableSequence): IColorableSequence;
+  // (undocumented)
+  static whiteBackground(text: string | IColorableSequence): IColorableSequence;
+  // (undocumented)
+  static yellow(text: string | IColorableSequence): IColorableSequence;
+  // (undocumented)
```

Comment:
These aren't worth documenting, but you could explain how they work in the class comment.